deploy/role.yaml

```diff
@@ -105,6 +105,17 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Does Role also need access to the routes resource?

Reply:
We are using ClusterRole. There is another PR #37 for removing the Role resource altogether.